frame/system/src/lib.rs

```diff
@@ -944,53 +1017,51 @@ impl<T: Trait> Happened<T::AccountId> for CallKillAccount<T> {
 // Anything more complex will need more sophisticated logic.
 impl<T: Trait> StoredMap<T::AccountId, T::AccountData> for Module<T> {
 	fn get(k: &T::AccountId) -> T::AccountData {
-		Account::<T>::get(k).1
+		Account::<T>::get(k).data
 	}
 	fn is_explicit(k: &T::AccountId) -> bool {
 		Account::<T>::contains_key(k)
 	}
-	fn insert(k: &T::AccountId, t: T::AccountData) {
+	fn insert(k: &T::AccountId, data: T::AccountData) {
 		let existed = Account::<T>::contains_key(k);
-		Account::<T>::insert(k, (T::Index::default(), t));
+		Account::<T>::insert(k, AccountInfo{ nonce: Default::default(), refcount: 0, data });
```

Comment:
Why is refcount returning to zero here?

The goal is just to insert some account data onto the user?

We should use the existing ref count...

Reply:
that's the goal, yes.

Reply:
this looks indeed bad. good catch 👍

Reply:
```suggestion
		Account::<T>::mutate(k, |a| a.data = data);
```

This is what you want right?

Reply:
I am going to commit this suggestion.